pkg/sql/logictest/testdata/logic_test/sequences

```diff
@@ -2029,3 +2029,19 @@ CREATE SEQUENCE seqas_error
   INCREMENT BY 1
   MAXVALUE 123456
   CACHE 1
+
+subtest serial_normalization_type_name_conflict
+
+statement ok
+set serial_normalization = 'sql_sequence';
+create type t72820_i_seq AS enum ('a');
+create table t72820 (i SERIAL PRIMARY KEY)
+
+query T
+SELECT create_statement FROM [SHOW CREATE TABLE t72820]
+----
+CREATE TABLE public.t72820 (
+   i INT8 NOT NULL DEFAULT nextval('public.t72820_i_seq1'::REGCLASS),
```

Comment:
CI is failing because it's expecting an INT4. Are sequence values really 32-bit integers? I'd be mildly concerned if that were the case, naively it seems to me like we'd want to support tables with more than 4 billion rows at some point.

Reply:
It failed because somewhere we randomize `default_int_size`. Fixed by setting it explicitly.

Reply:
That's a relief 👍